src/middlewared/middlewared/plugins/audit/audit.py

```diff
@@ -558,6 +558,7 @@ async def setup(self):
         try:
             await self.middleware.call('audit.update_audit_dataset', audit_config)
         except Exception:
+            await self.middleware.call('alert.oneshot_create', 'AuditSetup', None)
```

Comment:
When is this alert removed?

Reply:
I purposefully left it to be manually dismissed by the customer.